utils/firebase/firebase.js

```diff
@@ -14,7 +14,7 @@ const firebaseConfig = {
   authDomain: AUTH_DOMAIN,
   databaseURL: DATABASE_URL,
   projectId: PROJECT_ID,
-  storageBucket: "",
+  storageBucket: "kit-app-54bc0.appspot.com",
```

Comment:
What is this change for?

Reply:
Assuming this is for using firebase storage, but please confirm

Reply:
Oh we should probably make this an environment variable actually

Reply:
just made an environment variable for this: FIREBASE_STORAGE_URL